docs/patching-pega-deployment.md

```diff
@@ -0,0 +1,135 @@
+
+See the next section to properly reference these new images in your Pega Helm chart.
+
+## Applying a zero-downtime Pega Platform patch using Helm charts – 120 minutes
+
+To keep your system current with a patch, you apply a patch to the Pega Platform software in your existing deployment by using Helm, you must customize your existing `pega` Helm chart with the specific, required or settings. This includes referencing the latest Pega-provided Docker images that are available for your release.
+
+The Pega patch application process takes at most 120 minutes total.
+
+To complete a zero downtime patch, you must configure the following settings in your existing Pega configuration files for your Pega Platform deployment:
+
+- Specify action.execute: upgrade-deploy to invoke the zero-downtime patch process.
+- Specify the schema name or names that will be upgraded. Since the process involves migrating rules to and from each schema (jdbc.rulesSchema: "YOUR_RULES_SCHEMA" and jdbc.dataSchema: "YOUR_DATA_SCHEMA").
+
+- Ensure one of the following:
+  - You pushed the images for your patch to the same repository that you used for your installation repository and the credentials for your repository account are the same as those in your `pega` Helm chart.  
+  - You pushed the images for your patch to a new repository and you update the parameters with your credentials for this new repository account in your `pega` Helm chart.
+
+- Update the tagging details, including the version and date of your latest Pega-provided `platform/pega` Docker image, that you downloaded to support your patch.
+
+- For existing AKS and PKS deployments, provide reference details for the service host and https service port of the Kubernetes API server (this is not required for installations). For example, the Kubernetes master is running at `https://<service_host>:<https_service_port>`. For EKS and GKE deployments, you leave the existing text values (do not leave them blank).
+
+- Update the tagging details, including the version and date of your latest Pega-provided `platform/search` Docker image, that you downloaded for your patch.
+
+- In the installer section of the Helm chart, update the following: 
+  - Update the tagging details, including the version and date of your Pega-provided `platform/installer` Docker image, that you downloaded to support your patch.
+  - Specify an `out-of-place` upgrade to apply a patch using the zero-downtime patch process.
+  - Specify a temporary rules schema name that the process creates in your existing database to support the patch process within the quotes.
```

Comment:
I believe it should be called as` new rules schema` and not` temporary rules schema `for an out-of-place upgrade.

Reply:
done